desktop/main.js

```diff
@@ -174,6 +174,10 @@ main.restartAndUpdate = function (updateFilePath) {
 };
 main.minimizeApp = function (menuItemLabels) {
     let imagePath;
+    // a workaround to correctly restore focus on windows platform
+    if (process.platform === 'win32') {
```

Comment:
What exactly is broken?

Reply:
on windows focus is not returned to previous field unless this workaround is added, e.g. if you are in a field on some page, open keeweb, copy (which does minimize), the focus is not returned to the field where you were before

Reply:
Please add this knowledge to the comment 😉